### ISSUE TITLE
How do I build a dll and pdb?

### ISSUE THREAD

**Issue by ecke101** [ifcquery/ifcplusplus#206]
When I'm building (vs2019), I get lots of errors of this type:
`1>C:\Users\erik\git\ifcplusplus\ifcplusplus\IfcPlusPlus\src\ifcpp\reader\ReaderSTEP.cpp(234,1): warning C4273: 'ReaderSTEP::removeComments': inconsistent dll linkage`
When build is finished I'm left with a 2GB file called `IfcPlusPlusrd.lib`

edit: Ok I missed the obvious to change from RelWithDebugInfo to Release. Then the .dll is generated. But what if I need the pdb file to do profiling of the code?

**Comment by ifcquery** [comment]
IFCQUERY_LIB needs to be defined in the project settings. If it is removed, there will be inconsistent dll linkage errors.

**Comment by r-silveira** [comment]
I downloaded the last snapshot and compiled it in vs2019 - Version 16.9.0 (last update) with the project's default configuration in release mode.
I got a .lib and .dll file. When I run my program, it says that .dll is not for windows or is corrupted. In a hexadecimal editor, the dll file is filled with zeros (~2MB).
If I change the option to generate a static library, I get a .lib with ~3GB and when I use it, I get some link errors.

My preprocessor definitions are:
UNICODE
WIN32
_JANELAS
NDEBUG
_MBCS
ENABLE_OPENMP
IFCQUERY_AS_DYNAMIC_LIBRARY
IFCQUERY_LIB

I'm forgetting something?
